### PR TITLE
Add settings for AWS keys

### PR DESCRIPTION
**Pull request by grantmcconnaughey**
This PR adds two Django settings: `COGNITO_AWS_ACCESS_KEY_ID` and `COGNITO_AWS_SECRET_ACCESS_KEY`

_This PR looks like I changed a lot of lines because my editor cleans up all trailing whitespace on save._

### CODE REVIEW

File: warrant/django/backend.py [hunk 1]
```diff
@@ -67,7 +67,10 @@ def authenticate(self, username=None, password=None):
         :return: returns User instance of AUTH_USER_MODEL or None
         """
         cognito_user = CognitoUser(
-            settings.COGNITO_USER_POOL_ID,settings.COGNITO_APP_ID,
+            settings.COGNITO_USER_POOL_ID,
+            settings.COGNITO_APP_ID,
+            access_key=getattr(settings, 'COGNITO_AWS_ACCESS_KEY_ID', None),
```
Comment (bjinwright) [inline warrant/django/backend.py]: I'd like to make the settings AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY like most Django project so if you use the default environment variables it isn't so confusing.

Comment (bjinwright) [inline warrant/django/backend.py]: After you change the names if you could use `envs.env` to pull in the environment variables as settings in the `cdu.settings` file I think that would fix the failing tests.

**Comment by grantmcconnaughey** [comment]
@bjinwright All right, all fixed up. Thanks for the quick feedback!

**Comment by grantmcconnaughey** [comment]
Hmm, all the PRs I've submitted have failed for the same reason:

```
botocore.exceptions.NoRegionError: You must specify a region.
```

**Comment by bjinwright** [comment]
Nope I'm not getting that error on one of my feature that has the same Django backend code. [Feature Branch Build](https://travis-ci.org/capless/warrant/builds/223170488)

**Comment by grantmcconnaughey** [comment]
Any idea why it would be failing? I'd be happy to try to fix it if you can point me in the right direction.

Also, what would you think about adding a `get_cognito_user` method to that backend class? That way users can subclass `CognitoBackend` and only need to override `get_cognito_user` to customize how that object gets created. For example, I would like use [django-constance](https://django-constance.readthedocs.io/en/latest/) to store the user pool ID and cognito app ID. Currently I have to copy and paste the entire `authenticate` method code to change 2 lines.

**Comment by bjinwright** [comment]
I refactored the AWS IAM policy that runs on TravisCI and that still didn't do that trick. I am afraid we'll need to refactor the code to use pass in the region as an argument.

**Comment by bjinwright** [comment]
I also ran the `develop` branch build again and it passed so adding the region to the instantiation is the move.

**Pull request merged**